packages/venia-ui/lib/components/CartPage/PriceSummary/index.js

```diff
@@ -1 +1 @@
-export { default, PriceSummaryQuery } from './priceSummary';
+export { default, GET_PRICE_SUMMARY } from './priceSummary';
```

Comment:
Ah I meant to actually remove this export from `priceSummary.js`. Can you just remove the export from line 102 of that file, move the query _above_ the component, and then remove the export here? Thanks!

Reply:
Ok, I removed unused export and moved query above component.